agent/containers/images/visualizers/README.md

```diff
@@ -1,12 +1,13 @@
+NOTE: Source code moved to a new repo: [distributed-system-analysis/visualizers](https://github.com/distributed-system-analysis/visualizers)
```

Comment:
Let's remove this file entirely, and move this one line to the `agent/container/images/README.md`.